source/player.js

```diff
@@ -0,0 +1,27 @@
+var Pixi = require("pixi.js")
+var Keyb = require("keyb")
+
+export default class Player extends Pixi.Sprite {
```

Comment:
I always like to name the file to match what it is exporting. So for example:

If we're exporting a class (which we usually like to capitalize), such as `Player`, we should also name the file with a capital letter, like `Player.js`.

Also, just my preference, but I think we should name it Junkership, not Player. :p